tests/dummy/app/pods/demo/controller.js

```diff
@@ -1,10 +1,10 @@
 import Ember from 'ember'
 
-const {Controller} = Ember
+const {A, Controller} = Ember
 
 export default Controller.extend({
 
-  tableSortList: [
+  tableSortList: A([
```

Comment:
would this affect consuming apps? Do they have to change any sort lists they use into Ember Arrays?